packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx

```diff
@@ -0,0 +1,144 @@
+import { useTranslation } from 'react-i18next';
+import {
+  Box,
+  Button,
+  Checkbox,
+  FormControl,
+  InputLabel,
+  ListItemText,
+  MenuItem,
+  OutlinedInput,
+  Select,
+  SelectChangeEvent,
+  Stack,
+  Typography,
+} from '@mui/material';
+import { CustomDropdown } from '@pagopa-pn/pn-commons';
+import SearchIcon from '@mui/icons-material/Search';
+import { useState } from 'react';
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250,
+    },
+  },
+};
+const names = [
+  'Oliver Hansen',
+  'Van Henry',
+  'April Tucker',
+  'Ralph Hubbard',
+  'Omar Alexander',
+  'Carlos Abbott',
+  'Miriam Wagner',
+  'Bradley Wilkerson',
+  'Virginia Andrews',
+  'Kelly Snyder',
+];
+const names2 = [
+  'Oliver Hansen',
+  'Van Henry',
+  'April Tucker',
+  'Ralph Hubbard',
+  'Omar Alexander',
+  'Carlos Abbott',
+  'Miriam Wagner',
+  'Bradley Wilkerson',
+  'Virginia Andrews',
+  'Kelly Snyder',
+];
+
+const ProxiesChargedCompany = () => {
+  const { t } = useTranslation(['deleghe']);
+  const rows = false;
+  const [personName, setPersonName] = useState<Array<string>>([]);
```

Comment:
Remove the code that for now is useless. At the moment we don't need the state. Focus on the appearance of the component

Reply:
done